nixos/tests/sddm.nix

```diff
@@ -65,5 +66,99 @@ let
       '';
     };
   };
+
+
+
+  /*
+  Tests using different shell dot profile files
+  to initialize the graphical session
+  key = shell name
+  value = list of profile files
+          these files will be created in user's home directory
+          each will be prefixed with a "."
+  */
+  profileTestConfigs = {
+    bash = [ "bash_profile" "bash_login" "profile" ];
+    #zsh = [ "zprofile" ];
```

Comment:
Why is this commented?

Reply:
Because of the [TODO](https://github.com/NixOS/nixpkgs/pull/187620/files#diff-b8e8944f0cb9e102b2bccd7313bc864b1b323d6fde5756c4a0f644a25829d481R87) later on. I could remove the line if so wished or add a comment pointing to the TODO. It's unclear to me what would be better.

I could also create a new issue and add the issue number to the related lines. What do you think?